packages/lib/src/link/Link.tsx

```diff
@@ -78,44 +78,37 @@ const StyledLink = styled.div<{
   background: none;
   border: none;
   padding: 0;
+  ${(props) => `padding-bottom: ${props.theme.underlineSpacing};`}
   cursor: pointer;
   font-size: ${(props) => props.theme.fontSize};
   font-weight: ${(props) => props.theme.fontWeight};
   font-style: ${(props) => props.theme.fontStyle};
   font-family: ${(props) => props.theme.fontFamily};
-  text-decoration-color: transparent;
+  text-decoration: none;
   width: fit-content;
-  ${(props) =>
-    `padding-bottom: ${props.theme.underlineSpacing};
-     border-bottom: ${props.theme.underlineThickness} ${props.theme.underlineStyle} transparent;`}
   ${(props) => props.disabled && "cursor: default;"}
   color: ${(props) =>
     props.inheritColor ? "inherit" : !props.disabled ? props.theme.fontColor : props.theme.disabledFontColor};
   ${(props) => (props.disabled ? "pointer-events: none;" : "")}
   &:visited {
     color: ${(props) => (!props.inheritColor && !props.disabled ? props.theme.visitedFontColor : "")};
-    &:hover {
-      ${(props) =>
-        `color: ${props.theme.visitedFontColor};
-         border-bottom-color: ${props.theme.visitedUnderlineColor};`}
+    & > span:hover {
+      ${(props) => `color: ${props.theme.visitedFontColor};
+                    border-bottom-color: ${props.theme.visitedUnderlineColor};`}
     }
   }
-  &:hover {
-    ${(props) =>
-      `color: ${props.theme.hoverFontColor};
-       border-bottom-color: ${props.theme.hoverUnderlineColor};
-       cursor: pointer;`}
+  &:active {
```

Comment:
We can move the `:active` selector to the `LinkContainer` styled component and remove the `& > span` selector.

Reply:
AFAIK, :active selector only works for <a> or <button> (the same as :visited), so I don't think it is suitable for this case.

Reply:
Did you try? I tested it and it worked fine.